### PR TITLE
refactor/1072 - Overhaul of KELP

### LINKED ISSUE #1072: Re-Introduce El Paso International Airport (KELP)
The purpose of this issue is to: Update, and Re-enable El Paso International Airport.

### PR DESCRIPTION
**Pull request by JPmAn24**
Resolves #1072 

The purpose of this pull request is to Update and Re-introduce KELP into the game.

### CODE REVIEW

**Comment by erikquinn** [comment]
@jpman24 I see the `WIP` label-- is this still a work in progress, or is it ready for review?

**Comment by JPmAn24** [comment]
@erikquinn Sorry, fixed. It's ready for review now.

File: assets/airports/kelp.json [hunk 1]
```diff
@@ -11,11 +11,8 @@
     "ctr_ceiling": 17000,
     "initial_alt": 7000,
     "position": ["N31.807327503657113", "W106.37640380404419", "3962ft"],
-    "rangeRings": {
-        "enabled": true,
-        "center": [31.807327503657113, -106.37640380404419],
-        "radius_nm": 5.0
-    },
+    "rr_radius_nm": 5.0,
```
Comment (Fechulo) [inline assets/airports/kelp.json]: You should use the new range ring format here

**Comment by Fechulo** [comment]
Also please add the AIRAC tag at the top :smile:

**Comment by JPmAn24** [comment]
@Fechulo Done!

**Comment by erikquinn** [comment]
Had some merge problems with this last night. Weird stuff in b04ba9f causing the issue. May need to squash to a single commit to get it in. I'll try again hopefully later tonight after work.

**Comment by erikquinn** [comment]
Squashed and merged!

**Pull request merged**